hexa/pipelines/tests/test_schema.py

```diff
@@ -516,3 +516,39 @@ def test_delete_pipeline_version(self):
             {"success": True, "errors": []},
             r["data"]["deletePipelineVersion"],
         )
+
+    def test_add_pipeline_recipients(self):
```

Comment:
Could you try to test the case where you remove members of the pipeline and where you update the recipients with new recipients and some recipients removed as well ?

Reply:
And also check if the mail has been sent

Reply:
The email will be sent when the pipeline has finished running.

Reply:
Oh yes. But you can still check what happens when `mail_run_recipients(run)` is called.

Reply:
And I'd keep at least one recipient in the recipients's list during the update to check if the deletion of the recipient not in the list works.

Reply:
Already done for the deletion 
```python
def test_update_pipeline_recipients(...)
 ```

Reply:
Yes but could you test with more than one recipient in the list ? It feels like we are half testing it 🤷🏻